document/toast/toast.css

```diff
@@ -0,0 +1,16 @@
+.grid-container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  grid-template-areas:
+    "header header header"
+    "side toast toast";
+}
+
+#toast-section {
+  grid-area: toast;
+}
+
```

Comment:
Better would be to use class file as a good practice. I know you are using it once but still if you used it as class it would help in many ways, like while debugging people would know that you have used class file. It's a suggestion tough.